edx_toggles/toggles/internal/waffle/legacy.py

```diff
@@ -32,14 +32,15 @@
 from .switch import WaffleSwitch as NewWaffleSwitch
 
 
-class LegacyToggleMonitoringMixin():
+class LegacyToggleMonitoringMixin:
```

Comment:
I hope the Python3/black changes will be coming soon so they don't have to drip in for every PR.  They just distract from what I actually care about.

Reply:
Yes, sorry about that.